build-system/tasks/helpers.js

```diff
@@ -628,7 +646,7 @@ async function compileJs(srcDir, srcFilename, destDir, options) {
    */
   async function doCompileJs(options) {
     const buildResult =
-      options.minify && shouldUseClosure()
+      options.minify && shouldUseClosure(options)
```

Comment:
```suggestion
      shouldUseClosure(options)
```

Reply:
This would break this usage:
https://github.com/ampproject/amphtml/blob/debee40a008fd5fe0397a237e6c9be5f46cbdcdf/build-system/tasks/check-sourcemaps.js#L121